### PR TITLE
feat: implement report sharing

### PR DESCRIPTION
**Pull request by jaredsavage**
Moves results into info page, adds facebook and twitter share buttons (might need tweaking)

### CODE REVIEW

File: src/components/share-results.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    width: 70vw;
+    max-width: 320px;
+  }
+`
+
+const FacebookShareButton = styled(ShareButton)`
+  background: #3b5998;
+`
+
+const TwitterSharebutton = styled(FacebookShareButton)`
```
Comment (vicrep) [inline src/components/share-results.tsx]: Should inherit ShareButton instead

File: src/components/share-results.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+  const url = encodeURI(window.location.href)
+  const facebookHref = `https://www.facebook.com/sharer/sharer.php?u=${url}&amp;src=sdkpreparse`
+  const twitterHref = `https://twitter.com/intent/tweet?text=${i18n.t(
+    'share.twitterShareText',
+    { url }
+  )}`
+
+  return (
+    props.classes && (
```
Comment (vicrep) [inline src/components/share-results.tsx]: Should check if length > 0 too?

File: src/i18n/translation.fr.js [hunk 3]
```diff
@@ -5,6 +5,12 @@ export default {
     changeAudience: 'Refaire le questionnaire',
     audiencePrefix: "Les informations ci-dessous s'appliquent à "
   },
+  share: {
+    viewResults: 'Voir vos résultats partageables',
```
Comment (vicrep) [inline src/i18n/translation.fr.js]: Personlisé? I feel like that's the feature we wanna advertise, not the shareability

File: src/components/share-results.tsx (hunk 1 above)
Comment (jaredsavage) [reply src/components/share-results.tsx]: 😂

File: src/components/share-results.tsx (hunk 2 above)
Comment (jaredsavage) [reply src/components/share-results.tsx]: 👌

File: src/i18n/translation.fr.js (hunk 3 above)
Comment (jaredsavage) [reply src/i18n/translation.fr.js]: cc @vanessajones

**Pull request merged**